### PR TITLE
Style pages and fix minor bugs

### PR DESCRIPTION
**Pull request by alternadiva**

### CODE REVIEW

File: components/Groups/AddNewGroupButton.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
   const router = useRouter();
 
   function handleRedirect() {
     router.push("/groups/add-group");
   }
-  return <button onClick={handleRedirect}>Add new group</button>;
+  return (
+    <div className="add-button-container">
+      <button onClick={handleRedirect} className="add-new-group-button">
+        <BsPlusCircle className="add-new-icon" />
```
Comment (sonianb) [inline components/Groups/AddNewGroupButton.js:13]: Should we add an aria-label to that one?

**Review by AsmahanM — APPROVED** [review]

File: components/Groups/AddNewGroupButton.js (hunk 1 above)
Comment (alternadiva) [reply components/Groups/AddNewGroupButton.js:13]: Good call, we should!

**Review by minju25kim — APPROVED** [review]
it looks amazing1

File: components/Groups/GroupsForm.js [hunk 2]
```diff
@@ -56,42 +58,65 @@ function GroupsForm() {
...
-        <input
-          type="file"
-          accept="image/png, image/jpeg, image/jpg"
-          id="group-avatar"
-          name="group-avatar"
-          onChange={handleAvatarChange}
-          required
-        />
-        <img src={avatarPreview} alt="preview" />
+    <main className="upload-section">
```
Comment (sonianb) [inline components/Groups/GroupsForm.js:61]: This should probably be a `<section>` ?

File: components/Products/ProductUpload.js [hunk 3]
```diff
@@ -72,11 +84,26 @@ function ProductUpload(props) {
...
+  function backToGroups() {
+    localStorage.removeItem("group");
+  }
+
   return (
-    <div>
-      <form encType="multipart/form-data" onSubmit={handleProductSubmit}>
-        <div>
-          <h2>Select a category</h2>
+    <main className="upload-section">
```
Comment (sonianb) [inline components/Products/ProductUpload.js:92]: This should probably be a `<section>` too?

File: components/Products/ProductUpload.js [hunk 4]
```diff
@@ -106,63 +133,62 @@ function ProductUpload(props) {
...
-          <input
-            type="text"
+        <div className="form-div">
+          <label htmlFor="item-description">Description*</label>
+          <textarea
             placeholder="Tell us about this item."
             id="item-description"
             name="item-description"
             ref={description}
+            required
```
Comment (sonianb) [inline components/Products/ProductUpload.js:154]: Nice addition to make the fields required ✨

**Review by sonianb — APPROVED** [review]
Great work ✨

**Pull request merged**